packages/courDeCassation/src/connector/mapper/extractors/extractRoute/extractRoute.ts

```diff
@@ -27,6 +28,10 @@ function extractRoute(
   switch (source) {
     case 'jurinet':
       try {
+        logger.error({
```

Comment:
```suggestion
        logger.log({
```
c'est pas encore une erreur a ce niveau la ;)

Reply:
C'est un logger qui doit être supprimer, j'ai du l'oublier